activities.json

```
@@ -351,6 +351,18 @@
     "title": "Cookie Store API",
     "url": "https://wicg.github.io/cookie-store/"
   },
+  {
+    "ciuName": null,
+    "description": "Defines the Partitioned cookierattribute. This attribute will indicate to user agents that these cross-site cookies should only be available in the same top-level context that the cookie was created in.",
+    "id": "chips",
+    "mozBugUrl": null,
+    "mozPosition": "positive",
+    "mozPositionDetail": "This spec provides an opt-in mechanism to enable partitioned cookies. This allows more browsers to support partitioned cookies, and once adopted by sites can facilitate the default blocking of third-party cookies. Importantly to us, the behavior of a cookie when the Partitioned attribute is not provided is left up to the implementer. This allows us to continue deployment of partitioning by default while not forcing other browsers to change their default cookie behavior immediately.",
```

```suggestion
    "mozPositionDetail": "This spec provides an opt-in mechanism to enable partitioned cookies. This allows more browsers to support partitioned cookies, and once adopted by sites can facilitate the default blocking of third-party cookies.",
```

That part is best left to a comment on the issue, I think.